### PR TITLE
Route deprecated API alerts to null

### PR DESCRIPTION
**Pull request by cblecker**
ref: https://issues.redhat.com/browse/OSD-2382

### CODE REVIEW

**Comment by cblecker** [comment]
Added SOP for this alert too: https://github.com/openshift/ops-sop/pull/679

File: pkg/controller/secret/secret_controller.go [hunk 1]
```diff
@@ -284,27 +287,8 @@ func addPDSecretToAlertManagerConfig(r *ReconcileSecret, request *reconcile.Requ
...
-		routes = append(routes,
-			&alertmanager.Route{
-				Receiver: "null",
-				Match: map[string]string{
-					"alertname": alert,
-				},
-			})
-	}
+	// generate the routes
+	routes := generateRoutes(alertsRouteWarning, alertsRouteNull)
```
Comment (dak1n1) [inline pkg/controller/secret/secret_controller.go]: Nice! This is much cleaner.

**Comment by dak1n1** [comment]
Thanks for adding the docs too!

**Pull request merged**